nixos/modules/virtualisation/linode-config.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+with lib;
+{
+  imports = [ ../profiles/qemu-guest.nix ];
+
+  ### Allow Root logins only using SSH keys
+  services.openssh = {
+    enable = true;
+
+    permitRootLogin = "prohibit-password";
+    passwordAuthentication = mkDefault false;
+  };
+
+  ### Set networking config
+  #
+  networking = {
+    usePredictableInterfaceNames = false;
```

Comment:
We are sure we want to set this?

Reply:
Linode recommends turning it off and notes that the problem predictable interfaces tries to solve is irrelevant on their platform with only a single interface, but they do also state that turning it off is optional: https://www.linode.com/docs/guides/install-nixos-on-linode/#disable-predictable-interface-names

The Azure and GCP/GCE image generators for NixOS both turn off predictable names as well as the qemu-vm generator so it may be something users would expect (but I don't know enough to say for sure)